src/silx/gui/widgets/FrameBrowser.py

```diff
@@ -360,53 +399,37 @@ def value(self):
         """Get selected value"""
         return self._slider.value()
     
-    def spinBoxFrameRate(self) -> qt.QSpinBox:
-        """Returns the SpinBox widget for FrameRate display."""
-        return self._menuPlaySlider._framerate_action._spinbox
-
-    def spinBoxInterval(self) -> qt.QSpinBox:
-        """Returns the SpinBox widget for interval display."""
-        return self._menuPlaySlider._interval_action._spinbox
-
     def setFrameRate(self, value: int):
-        """Set the FrameRate value for the PlaySlider"""
-        self.spinBoxFrameRate().setValue(value)
+        """Set the frame rate at which images are displayed"""
+        self._menuPlaySlider.setFrameRate(value)
         
     def getFrameRate(self) -> int:
-        """Returns the value from the FrameRate SpinBox."""
-        return int(self.spinBoxFrameRate().value())
+        """Get the frame rate at which images are displayed"""
+        return self._menuPlaySlider.getFrameRate()
     
-    def setInterval(self, value: int):
-        """Set the Interval value for the PlaySlider"""
-        self.spinBoxInterval().setValue(value)
+    def setPlayImageStep(self, value: int):
```

Comment:
Proposition to replace `Interval` which is quite generic (e.g., `QTimer.setInterval` which set the time interval) with something more specific.
Any better proposition welcomed